.ci/build_dattobd.gitlab-ci.yml

```diff
@@ -0,0 +1,121 @@
+#
+# Build dattobd on RHEL9
+#
+build:dattobd:on:rhel9:
+  stage: build dattobd
+  tags:
+    - docker-throttled
+  needs: ["create:rhel9:vm"]
+  script:
+    - apt-get update -y
+    - apt-get install sshpass -y
+    - sshpass -p "$VM_PASSWORD" ssh -o StrictHostKeyChecking=no datto@$RHEL9_VM_IP '
+        sudo rm -rf dattobd;
+        git clone https://github.com/krzysztof-wasieczko/dattobd.git --recursive;
+        cd dattobd;
+        ls;
+        sudo yum install kernel-devel-$(uname -r) -y;
+        sudo yum groupinstall "Development Tools" -y;
+        sudo make'
+
+#
+# Build dattobd on DEBIAN11
+#
+build:dattobd:on:debian11:
+# Known issue: some packages are missing on Debian11,
+# so building dattobd on debian is disabled
+  stage: build dattobd
+  tags:
+    - docker-throttled
+  needs: ["create:debian11:vm"]
+  script:
+    - apt-get update -y
+    - apt-get install sshpass -y
+    - sshpass -p "$VM_PASSWORD" ssh -o StrictHostKeyChecking=no datto@$DEBIAN11_VM_IP '
+        sudo rm -rf dattobd;
+        git clone https://github.com/krzysztof-wasieczko/dattobd.git --recursive;
+        cd dattobd;
+        ls'
+#        sudo yum install kernel-devel-$(uname -r) -y;
+#        sudo yum groupinstall "Development Tools" -y;
+#        sudo make'
+
+#
+# Build dattobd on UBUNTU1804
+#
+build:dattobd:on:ubuntu1804:
+  stage: build dattobd
+  tags:
+    - docker-throttled
+  needs: ["create:ubuntu1804:vm"]
+  script:
+    - apt-get update -y
+    - apt-get install sshpass -y
+    - sshpass -p "$VM_PASSWORD" ssh -o StrictHostKeyChecking=no datto@$UBUNTU1804_VM_IP '
+        sudo rm -rf dattobd;
+        git clone https://github.com/krzysztof-wasieczko/dattobd.git --recursive;
+        cd dattobd;
+        ls;
+        sudo yum install kernel-devel-$(uname -r) -y;
```

Comment:
yum won't work on Ubuntu :)

Reply:
And it doesn't even fail a job in GitLab pipelines :) Removed, looks like ubuntu doesn't need any additional installations